keras_rcnn/layers/pooling.py

```diff
@@ -55,3 +57,48 @@ def call(self, x, **kwargs):
 
     def compute_output_shape(self, input_shape):
         return None, self.regions, self.size, self.size, self.channels
+
+
+class ROIAlign(ROI):
+    """ROIAlign pooling layer proposed in Mask R-CNN (Kaiming He et. al.).
+
+    # Parameters
+    size: Fixed size [h, w], e.g. [7, 7], for the output slices.
+    regions: Integer, number of regions of interest.
+    stride: Integer, pooling stride.
+
+    # Returns
+    4D Tensor (number of regions, slice_height, slice_width, channels)
+    """
+
+    def __init__(self, size, regions, stride=1, **kwargs):
+        self.stride = stride
+        super(ROIAlign, self).__init__(size=size,
+                                       regions=regions,
+                                       **kwargs)
+
+    def call(self, x, **kwargs):
+        image, regions = x[0], x[1]
+
+        # convert regions from (x, y, w, h) to (x1, y1, x2, y2)
+        regions = keras.backend.cast(regions, keras.backend.floatx())
+        regions = regions / (self.stride + 0.0)
+        x1 = regions[:, 0]
+        y1 = regions[:, 1]
+        x2 = regions[:, 0] + regions[:, 2]
+        y2 = regions[:, 1] + regions[:, 3]
+
+        # normalize the boxes
+        shape = image._keras_shape
+        h = keras.backend.cast(shape[1], keras.backend.floatx())
+        w = keras.backend.cast(shape[2], keras.backend.floatx())
+        x1 /= w
+        y1 /= h
+        x2 /= w
+        y2 /= h
+        boxes = keras.backend.concatenate([y1[:, numpy.newaxis],
```

Comment:
Can you use `numpy.newaxis` here? I’d think you’d need to use `tensorflow.newaxis`.